src/static/js/ace.js

```diff
@@ -44,6 +44,7 @@ function scriptTag(source) {
 function Ace2Editor()
 {
   var ace2 = Ace2Editor;
+  var staticRoot = window.STATIC_ROOT_ADDRESS;
```

Comment:
would break iframes?

Reply:
No, All front-end tests are accepted without any failures.

Reply:
That's how I know this is gonna break things.  Your front end tests are < 50 lines of code :D.  You gotta include test coverage to know for sure that there are no failures.